agent/config/builder.go

```diff
@@ -962,7 +965,7 @@ func (b *builder) build() (rt RuntimeConfig, err error) {
 		AutoEncryptAllowTLS:                    autoEncryptAllowTLS,
 		AutoConfig:                             autoConfig,
 		Cloud:                                  b.cloudConfigVal(c.Cloud),
-		ConnectEnabled:                         connectEnabled,
+		ConnectEnabled:                         boolValWithDefault(&connectEnabled, true),
```

Comment:
I think this should probably be in the `agent/config/default.go` file now that I think about it, because the connect enable flag is something that is set on both agents and servers.

Reply:
Do we want it to be enable even when `autoEncryptAllowTLS || autoConfig.Enabled` is false?

Reply:
Connect can function without auto encrypt or auto config, so I think it should be okay.